Components (#184)
decidim/documentation: docs/en/modules/admin/pages/components/meetings/polls.adoc

= Polls

== Understand the basics

> Meetings polls allow administrators to define questions prior to an online meeting, that participants to the meeting can answer 

Reviewer: ```suggestion
Meetings polls allow administrators to define questions for a meeting, that participants can answer 
```
